### PR TITLE
Adding a new feed of clalit.

### PR DESCRIPTION
**Pull request by Eran1801**

### CODE REVIEW

File: libs/feed/clalit_feed.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+class ClalitFeed(FeedAbstract, ABC):
+
+    def pull(self) -> list[Document]:
+
+        documents: list[Document] = []
+        
+        url = 'https://apps.clalit.co.il/SupplierDataRESTService/api/supplier/GetJsonGroupSearchBody'
+
+        body_data = {
+            "serviceID": "33-0",
```
Comment (liranzxc) [inline libs/feed/clalit_feed.py:19]: Great job , there are more service id's that we can used ?

**Review by liranzxc — CHANGES_REQUESTED** [review]

File: apps/crawler-feeds/src/app.py [hunk 2]
```diff
@@ -41,7 +42,8 @@ def main():
     otef_lev_feed = OtefLevFeed()
     minster_of_health_mental_clinic = MOH_MentalHealthClinicsFeed()
     maccabi_feed = MaccabiTherapyClinicsFeed()
-
+    clalit_feed = ClalitFeed()
```
Comment (liranzxc) [inline apps/crawler-feeds/src/app.py:45]: I think we should pass args in the feed and pull many services of clalit

**Review by liranzxc — CHANGES_REQUESTED** [review]

File: apps/crawler-feeds/src/app.py (hunk 2 above)
Comment (Eran1801) [reply apps/crawler-feeds/src/app.py:45]: Good thinking.
Will be done in new assignments in the future.

File: libs/feed/clalit_feed.py (hunk 1 above)
Comment (Eran1801) [reply libs/feed/clalit_feed.py:19]: To check this it needed more research.
For this url : https://mushlam.clalit.co.il/he/development/Pages/development_3-10_years.aspx?tab=5
there is 1209 records, all this records will get in to our db.

**Review by liranzxc — APPROVED** [review]

**Pull request merged**